components/molecule/notification/src/_settings.scss

```diff
@@ -2,6 +2,7 @@ $w-notification: 100% !default;
 $z-notification: 1000 !default;
 $mh-notification: 500px !default;
 $sz-notification-icon: $sz-icon-m !default;
+$bdrs-notification: 8px !default;
```

Comment:
Taking advantage of this PR, we should review `border-radius` tokens with @jcger in the next SUI meeting, because in INMHO we should extend the tokens for border-radius property.

Reply:
Agree. in the past we've got `$bdrs-base`
https://github.com/SUI-Components/sui-theme/blob/master/src/settings-compat-v7/_box-style.scss#L4
but now is deprecated.

Reply:
I do agree with the tokens for Border Radius.
I suggest we continue with this PR as it is, and we move the border-radius in parallel